containers/images/foreman/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM centos:7
+
+RUN echo "tsflags=nodocs" >> /etc/yum.conf && \
+    yum -y install epel-release \
+    							 http://yum.theforeman.org/releases/nightly/el7/x86_64/foreman-release.rpm \
+    							 https://fedorapeople.org/groups/katello/releases/yum/nightly/katello/el7/x86_64/katello-repos-latest.rpm && \
+    yum -y install foreman-release-scl && \
+    yum -y install qpid-tools && \
+    yum -y install foreman tfm-rubygem-puma \
+    							 tfm-rubygem-katello tfm-rubygem-foreman_ansible tfm-rubygem-foreman_discovery tfm-rubygem-foreman_hooks \
+    							 tfm-rubygem-foreman_openscap tfm-rubygem-foreman_remote_execution tfm-rubygem-foreman_templates && \
+    yum clean all
+
+COPY container-assets/settings.yaml /etc/foreman/settings.yaml
+COPY container-assets/katello.yaml /etc/foreman/plugins/katello.yaml
+
+# Generate apipie:cache:index
+RUN yum -y install foreman-sqlite && \
+		yum clean all
+
+RUN DATABASE_URL=sqlite3::memory: foreman-rake db:create db:migrate apipie:cache:index
+
+RUN yum -y remove foreman-sqlite tfm-ror51-rubygem-sqlite3 && \
+		yum clean all
+# end generate apipie:cache:index
+
+RUN yum -y install tfm-rubygem-foreman_memcache && \
+		yum clean all
```

Comment:
Yes and no. I think some work needs to be done on `foreman_memcache` first. If this is included the build takes a lot longer and spits out a lot more debug info than is needed because it tries to connect to a memcache server that does not exist.